fuse.py

```diff
@@ -754,9 +838,6 @@ def _wrapper(func, *args, **kwargs):
 
         except BaseException as e:
             self.__critical_exception = e
-            log.critical(
```

Comment:
I'd prefer to keep this in, as this keeps the logging close to the point where the exception occurred. Waiting for FUSE to actually shut down, return the control to the calling python thread, which might then rethrow the exception (with a potentially messed-up stacktrace) bears to much potential for the exception being lost somewhere (i.e. the program crashing).